### PR TITLE
Feature/sw-1762-syncing-suspended-status-from-synergy

### PR DESCRIPTION
**Pull request by ALameLlama**

### CODE REVIEW

File: modules/servers/synergywholesale_hosting/synergywholesale_hosting.php [hunk 1]
```diff
@@ -246,10 +246,22 @@ function synergywholesale_hosting_synchronize($params)
...
+            'Pending Upgrade' => 'Active',
+            'Suspended' => 'Suspended',
+            'Suspended by Staff' => 'Suspended',
+            'Terminated' => 'Terminated',           
+        ];
 
         $updateData = [
             'username' => $apiResult->username,
             'domain' => $apiResult->domain,
+            'domainstatus' => $hostingStatus[$apiResult->status] ? $hostingStatus[$apiResult->status] : $params['status'],
```
Comment (Sn0wCrack) [inline modules/servers/synergywholesale_hosting/synergywholesale_hosting.php]:
```suggestion
            'domainstatus' => isset($hostingStatus[$apiResult->status]) ? $hostingStatus[$apiResult->status] : $params['status'],
```

**Review by Sn0wCrack — CHANGES_REQUESTED** [review]
Have one change I'd like you to make.

But can you also update the changelogs? New version will be 1.6.4

Thanks 😄

**Comment by ALameLlama** [comment]
I've updated the changelog :)

**Review by Sn0wCrack — APPROVED** [review]

**Comment by Luke-McMahon** [comment]
Sorry mate, would you be able to resolve the conflicts that are currently present?
We'll also have to increment the module version again in the changelogs (sorry) -- version will be 1.6.7

**Review by Luke-McMahon — APPROVED** [review]

**Pull request merged**